gameboard.js

```diff
@@ -0,0 +1,66 @@
+var Gameboard = function (size) {
+  if (size % 2 !== 0) {
+    size ++;
+  }
+  this.size = size;
+  this.piecesArray = new PiecesArray(Math.pow(size, 2)).shuffledPieces();
+  this.currentPieces = [];
+  this.matchedPieces = 0;
+};
+
+Gameboard.prototype.buildBoard = function(){
+  var index = 0;
+
+  for (numRow = 0; numRow < this.size; numRow++) {
+    $(".gameboard").append("<tr>");
+
+    for (numCell = 0; numCell < this.size; numCell++) {
+      var piece = this.piecesArray[index];
+      $(".gameboard").append("<td><div class='piece' data-piece-type='" + piece + "'>" + piece + "</div></td>");
+      index++;
+    }
+
+    $(".gameboard").append("</tr>");
```

Comment:
This isn't doing what you expect.

![screen shot 2015-05-12 at 3 59 50 am](https://cloud.githubusercontent.com/assets/86842/7583491/678e82a2-f85b-11e4-8324-2db95dc29c28.png)

jQuery tries to be clever and creates a `<tr>` element for this line and the one above. What you want to do is create the `<tr>` once, then append the `<td>`s to it.